### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by lexming** [comment]
This needs a version bump to v.0.5.2

File: lib/vsc/filesystem/oceanstor.py [hunk 1]
```diff
@@ -92,18 +91,19 @@
...
     """Client for OceanStor REST API"""
 
-    def __init__(self, *args, ssl_verify=True, **kwargs):
+    def __init__(self, *args, **kwargs):
         """Wrapper for Client.__init__() allowing to disable SSL certificate verification"""
         super(OceanStorClient, self).__init__(*args, **kwargs)
 
         # X-Auth-Token header
         self.x_auth_header = None
+        ssl_verify = kwargs.get('ssl_verify', False)
```
Comment (lexming) [inline lib/vsc/filesystem/oceanstor.py]: Better default to a proper SSL connection and verify the certs
```suggestion
        ssl_verify = kwargs.get('ssl_verify', True)
```

**Comment by lexming** [comment]
RIP my quotes :slightly_smiling_face:

**Comment by wpoely86** [comment]
It's not me, it's black not liking your quotes :grin:

**Review by lexming — APPROVED** [review]

**Pull request merged**